src/tasks/nep.ts

```diff
@@ -57,6 +64,10 @@ export const NEP_TASKS: BaggoTask[] = [
             .if_(
               "!hppercentbelow 75 && !mpbelow 40",
               Macro.trySkill($skill`Double Nanovision`).trySkill($skill`Double Nanovision`)
+            )
+            .externalIf(
+              get("spookyVHSTapeMonsterTurn", 0) <= totalTurnsPlayed() - 8,
```

Comment:
This will trigger even if the mall price is not economical if we have some already in our inventory.

Reply:
You'll probably just have to keep that behavior in your fork. Most users would expect inventory to not be used if mall price isn't economical as that is what garbo does.